helix-cli/src/docker.rs

```diff
@@ -88,19 +88,19 @@ impl<'a> DockerManager<'a> {
         Ok(output)
     }
 
-    /// Run a docker-compose command with proper project naming
+    /// Run a docker compose command with proper project naming
     fn run_compose_command(&self, instance_name: &str, args: Vec<&str>) -> Result<Output> {
         let workspace = self.project.instance_workspace(instance_name);
         let project_name = self.compose_project_name(instance_name);
 
         let mut full_args = vec!["--project-name", &project_name];
         full_args.extend(args);
 
-        let output = Command::new("docker-compose")
+        let output = Command::new("docker compose")
             .args(&full_args)
```

Comment:
compose is now an arg. Should be `Command::new("docker").arg("compose").args(&full_args)....`